test/views/authorised_to_view/AuthorisedToViewSearchSpec.scala

```diff
@@ -162,5 +168,14 @@ class AuthorisedToViewSearchSpec extends SpecBase with MustMatchers {
 
     val form: Form[String] = new EoriNumberFormProvider().apply()
     val date = "SomeDate"
+
+    val authorisedToViewInstance: authorised_to_view_search = app.injector.instanceOf[authorised_to_view_search]
+
+    val sampleView: Document = Jsoup.parse(authorisedToViewInstance.apply(
+        form = form,
```

Comment:
Parameters from line 175-178 are not formatted properly (not indented correctly). Please have a look.

Reply:
Discussed

Reply:
This has been addressed over the call